examples/todo-app-tutorial/package.json

```diff
@@ -15,7 +15,7 @@
     "@microsoft/fast-colors": "^5.0.12",
     "@microsoft/fast-element": "latest",
     "@microsoft/fast-foundation": "latest",
-    "lodash-es": "^4.17.15",
+    "lodash-es": "4.17.15",
```

Comment:
@radium-v why are we pinning this version?

Reply:
It's due to this bug (lodash/lodash#5033) in `4.17.20`. Pretty much everything breaks with that latest version.

An alternative to pinning could be setting a top-level resolution entry, but I don't know how that will affect nested dependencies.